src/EnergyPlus/VentilatedSlab.cc

```diff
@@ -648,27 +639,27 @@ namespace VentilatedSlab {
 
             // Low Air Temp :
 
-            state.dataVentilatedSlab->VentSlab(Item).HotAirLoTempSched = state.dataIPShortCut->cAlphaArgs(11);
-            state.dataVentilatedSlab->VentSlab(Item).HotAirLoTempSchedPtr = GetScheduleIndex(state, state.dataIPShortCut->cAlphaArgs(11));
-            if ((state.dataVentilatedSlab->VentSlab(Item).HotAirLoTempSchedPtr == 0) && (!lAlphaBlanks(11))) {
+            ventSlab.HotAirLoTempSched = state.dataIPShortCut->cAlphaArgs(11);
```

Comment:
And here.

Reply:
I'm confused about this one. With the other ones ([this](https://github.com/NREL/EnergyPlus/pull/9274#discussion_r809498400) and [this](https://github.com/NREL/EnergyPlus/pull/9274#discussion_r809498515)), I converted the `string` to `std::array<string_view>` and got the enum using `getEnumerationValue`.

Reply:
Got it :)